packages/chat-stateful-client/src/ChatContext.ts

```diff
@@ -416,3 +433,72 @@ const toChatError = (target: ChatErrorTarget, error: unknown): ChatError => {
   }
   return new ChatError(target, new Error(`${error}`));
 };
+/* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
+class MessageQueue {
```

Comment:
A more descriptive name could help here. `ResourceDownloadQueue`?

Might be something to break out into a separate file.